### ISSUE TITLE
/var/log or /var/log/shinken should be a docker volume

### ISSUE THREAD

**Issue by erny** [rohit01/docker_shinken#9]
I think, any logs / data stored should be persisted outside of the docker container, so that when rebuilding no data would be lost (much like custom_config).

**Comment by rohit01** [comment]
Agree. For simplicity, it is not written in docs. You can mount the /var/log directory of the container just like custom configs. Just add one more volume mounting. A quick example:

```
-v "/var/log/shinken:/var/log/shinken"
```
